src/components/Amount/Amount.tsx

```diff
@@ -52,6 +53,24 @@ type PercentAmountProps = AmountProps & {
   autoColor?: boolean
 }
 
+const truncateLargeNumber = (value: string, commaLimit: number = 5) => {
+  const [numberPart, currencyPart] = value.split(' ')
+
+  if (!numberPart) return value
+
+  const commas = numberPart.match(/,/g) || []
```

Comment:
This won't generalize to different fiat formats, I guess if we really wanted to, we could probably find a way to capture the delimiter, but there must be a better way.

Speaking out loud here, what about either:

- Using a very rough rounding using M or similar units (though for `4,211,817,743,761,223,513,624,031,901,408,326,070,532,995` that obviously won't generalize either so disregard me)
- Using a good ol' text trim instead of dealing with numbers (which at this point is effectively what we're doing i.e the current solution hides all the right part under `...` ellipsis)

Reply:
Removed in [ad9a073](https://github.com/shapeshift/web/pull/8254/commits/ad9a0730cf49232b194372c93b0814237de8af48)